packages/shared/prisma/migrations/20240917072611_remove_covered_indexes/migration.sql

```diff
@@ -0,0 +1,18 @@
+-- DropIndex
```

Comment:
@marcklingen @maxdeichmann Apparently, Prisma does dumb things and wraps this into a single transaction (https://github.com/prisma/prisma/issues/22922) 😞 
Could you review if you agree with the general premise of this change? If yes, I'll separate all those changes into separate folders to avoid the transactions in Prisma - I'd just prefer not to do this if we don't want to follow through with this change 😄

Reply:
Agree. AFAIK they need to be in a separate migration files as otherwise concurrently also doesn't work correctly.

Reply:
Agree as well.